### ISSUE TITLE
Short strands sequencing

### ISSUE THREAD

**Issue by evgisa** [lykaust15/DeepSimulator#25]
Hi, I wanted to know about the performance of DeepSimulator using short reference sequences (around 1500pb). Since the model was trained using longer sequences I was wondering if it would be able to simulate the sequencing of shorter reads or if the simulation in this case won't be accurate. Thank you for you time!

**Comment by lykaust15** [comment]
Hi, thank you very much for your interest!
Based on our experience, as for the context-dependent pore model, the signal length which would affect the simulation quality is around 300 (or 40bp). In fact, when we trained the model, we divided the long sequences into short chunks to obtain the well-trained pore model. For the short reference sequences (1500bp), I think it can still work fine.

As for the context-independent pore model, since it is a purely statistical model which is not related to the sequence length, it should have similar performance on long or short reference sequences.

Feel free to let us know your result and we are glad to help you further.

Sincerely,
Yu
